ecosystem/gstreamer_plugin/gst_mtl_st20p_tx.c

```diff
@@ -543,3 +571,101 @@ GST_PLUGIN_DEFINE(GST_VERSION_MAJOR, GST_VERSION_MINOR, mtl_st20p_tx,
                   "software-based solution designed for high-throughput transmission",
                   plugin_init, PACKAGE_VERSION, GST_LICENSE, GST_PACKAGE_NAME,
                   GST_PACKAGE_ORIGIN)
+
+static int gst_mtl_st20p_tx_frame_done(void* priv, struct st_frame* frame) {
+  if (frame == NULL || frame->opaque == NULL) {
+    return 0;
```

Comment:
Maybe print some error? It will be easy to miss if it returns 0 anyway.

Reply:
Well, do you want to have one GST_DEBUG() call per frame? it is quite a lot of information